src/test/scala/org/ergoplatform/api/routes/WalletApiRouteSpec.scala

```diff
@@ -29,8 +29,18 @@ class WalletApiRouteSpec extends FlatSpec
 
   implicit val paymentRequestEncoder: PaymentRequestEncoder = new PaymentRequestEncoder(ergoSettings)
   implicit val assetIssueRequestEncoder: AssetIssueRequestEncoder = new AssetIssueRequestEncoder(ergoSettings)
+  implicit val requestsHolderEncoder: RequestsHolderEncoder = new RequestsHolderEncoder(ergoSettings)
   implicit val ergoAddressEncoder: ErgoAddressEncoder = new ErgoAddressEncoder(ergoSettings)
 
+  it should "generate arbitrary transaction" in {
+    val request = PaymentRequest(Pay2SAddress(Values.FalseLeaf), 100L, None, None, 100000L)
+    val holder = RequestsHolder((0 to 10).map(_ => request), 10000L)
```

Comment:
Let's it have both PaymentRequest and AssetIssueRequest

Reply:
Fixed